bacon/Base.lproj/Main.storyboard

```diff
@@ -444,10 +515,13 @@
                         </constraints>
                         <viewLayoutGuide key="safeArea" id="2yH-Go-pk1"/>
                     </view>
+                    <connections>
+                        <outlet property="tableView" destination="Wxi-zi-XoU" id="0cE-oP-dMw"/>
+                    </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="uGh-dA-pXs" userLabel="First Responder" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="801" y="-558"/>
+            <point key="canvasLocation" x="800.78125" y="-558.3984375"/>
```

Comment:
Why did integers become floating point numbers? O.o

If these represent pixels, shouldn't they be strictly integers?